react/components/QuantitySelector.tsx

```diff
@@ -39,10 +52,10 @@ const validateDisplayValue = (value: string, maxValue: number) => {
   return `${normalizeValue(parsedValue, maxValue)}`
 }
 
-const getDropdownOptions = (maxValue: number) => {
+const getDropdownOptions = (maxValue: number, intl: InjectedIntl) => {
   const limit = Math.min(9, maxValue)
   const options = [
-    { value: 0, label: '0' },
+    { value: 0, label: `0 - ${intl.formatMessage(messages.remove)}` },
```

Comment:
I'd remove the 0, not sure it's useful.
```suggestion
    { value: 0, label: intl.formatMessage(messages.remove) },
```

Reply:
I asked our designer and we decided to keep the zero. The reason is that in a list containing just numbers, having "0" in the label makes it more clear that option is related to a quantity.